handlers/movies.go

```diff
@@ -0,0 +1,30 @@
+package handlers
+
+import (
+	"go_project/models"
+
+	"github.com/labstack/echo/v4"
+)
+
+func GetMovieById(c echo.Context) error {
```

Comment:
By convention, every exported stuff (function, variable, constant, etc) should have a description

```go
// GetMovieById - Description here....
func GetMovieById(c echo.Context) error {
```

Reply:
Done!